### PR TITLE
Changes for blog page

### PR DESCRIPTION
**Pull request by edwoodward**
Added featured_video field
```
"featured_video": [
        {
            "type": "video",
            "value": "https://youtu.be/ob7W93ZO4LE",
            "id": "e7babf3f-0bb8-45ee-aa24-b9eb80976f32"
        }
    ],
```
Added gated_content field

`"gated_content": false,`

Added CTA as part of blog post
```
{
            "type": "blog_cta",
            "value": {
                "heading": "Call To Action!!!",
                "description": "This is a call to action. Get after it!",
                "button_text": "Do it now!",
                "button_href": "https://openstax.org/",
                "alignment": "bottom"
            },
            "id": "35e35c36-a146-4322-8113-0bd65c21931f"
        }
```
blog-post-options

### CODE REVIEW

File: news/migrations/0041_alter_newsarticletag_tag.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('taggit', '0004_alter_taggeditem_content_type_alter_taggeditem_tag'),
```
Comment (mwvolo) [inline news/migrations/0041_alter_newsarticletag_tag.py:10]: did you manually change this or was it part of `makemigrations`?

**Review by mwvolo — APPROVED** [review]
there's a lot of migrations in the PR, any reason for that?

File: news/migrations/0041_alter_newsarticletag_tag.py (hunk 1 above)
Comment (edwoodward) [reply news/migrations/0041_alter_newsarticletag_tag.py:10]: It was part of the migrations

**Comment by edwoodward** [comment]
@mwvolo The Blog Body block is used in several locations. Adding the CTA block caused migrations in all of the other pages using the body block

**Pull request merged**